csharp/src/Ice/Communicator.cs

```diff
@@ -133,6 +106,48 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
         public int DefaultInvocationTimeout { get; }
         public int DefaultLocatorCacheTimeout { get; }
 
+        /// <summary>
```

Comment:
I prefer the more compact doc-comment style you deleted.

Reply:
Really? I find having it shoved in the content of the comment unpalatable personally; having it separated out makes the comment easier to read in my opinion.

Reply:
For me the compact style is nicer :)

Reply:
Is that just because it means 2 less lines of code? :)
Shall we put it to a vote, or should I just overrule myself?

Reply:
Microsofts preferred style seems to be the way we were doing it before.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments

Reply:
Yeah, in general they seem to prefer it Bernard-style, but they do it this way once here:
```
/// <summary>
///    Contains a list of integers.
/// </summary>
```
To be honest, the page seems kind of inconsistent with it's own style... But change it back I shall!

it also looks like they indent the following lines with 4 spaces some of the time
```
/// <summary>This method changes the point's location to
///    the given coordinates.</summary>
/// <see cref="Translate"/>
```
But also sometimes they don't.